File: c-sharp/Api/PremAddress.cs
using CsvHelper.Configuration.Attributes;

namespace Npb.Agview.Api.Example
{
    public class PremAddress
    {
        [Name("usda_pin")]
        public string UsdaPin { get; set; }

        [Name("street_address")]
        public string StreetAddress { get; set; }

        [Name("city")]
        public string City { get; set; }

        [Name("state")]
        public string State { get; set; }

        [Name("zip")]
        public string Zip { get; set; }
    }
}

File: java/src/main/java/org/npb/agview/api/example/PremAddress.java
package org.npb.agview.api.example;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode
public class PremAddress {
    private String usdaPin;
    private String streetAddress;
    private String city;
    private String state;
    private String zip;
}
